src/locales/en.json

```diff
@@ -77,7 +77,7 @@
     "group": "Group selection",
     "ungroup": "Ungroup selection",
     "collaborators": "Collaborators",
-    "gridMode": "Grid mode",
+    "gridMode": "Show grid",
```

Comment:
```suggestion
    "showGrid": "Show grid",
```

to keep it consistent

Reply:
Copy can generally be divorced from code variable naming.

Reply:
True.. but while we at it.. if we were about to introduce it today it would be `showGrid` :) and less confusing for new comers/